docs/02-app/02-api-reference/04-functions/unstable_noStore.mdx

```diff
@@ -36,6 +39,23 @@ export default async function Component() {
 }
 ```
 
+## Page Usage
+
+If you have an `async` `page.tsx` with PPR enabled you will need to declare `noStore()` to prevent PPR from attempting
+to make a static shell for a page that contains async calls outside the return statement. Note the `noStore()` must be
+declared outside the function at the file level prior to the export.
+
+```jsx filename="page.tsx"
+import { unstable_noStore as noStore } from 'next/cache';
+
+noStore();
+
+export default async function Page() {
```

Comment:
It might make more sense to show this pattern:

```js
import { unstable_noStore as noStore } from 'next/cache';
import { Suspense } from 'react';

async function ProductQuantity() {
  noStore();
  let res = await fetch('https://api.vercel.app/products/1');
  let data = await res.json();
  return <h1>{data.stock}</h1>;
}

export default function Page() {
  return (
    <section>
      <h1>Product</h1>
      <Suspense fallback={null}>
        <ProductQuantity />
      </Suspense>
    </section>
  );
}
```

Where the async component is a child of the page, since often there's _something_ you want to prerender (but not always).